app/core/Router.coffee

```diff
@@ -1,6 +1,28 @@
 locale = require 'locale/locale'
+normalizeViewRoute = (path) -> if not _.string.startsWith(path, 'views/') then "views/#{path}" else path
 
-go = (path, options) -> -> @routeDirectly path, arguments, options
+go = ({path, load}, options) ->
+  normedPath = normalizeViewRoute(path)
+  f = -> @routeDirectly path, arguments, options
+  _.assign(f, {path: normedPath, load})
+  return f
+  
+# views with multiple routes
+sharedViews = {
```

Comment:
Can we move this down to be right above `CocoRouter.routes`, so it will be in close proximity to all the unique paths?